ocf_data_sampler/load/site.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+import xarray as xr
+
+from ocf_data_sampler.config.model import Site
+
+
+def open_site(sites_config: Site) -> xr.DataArray:
+
+    # Load site generation xr.Dataset
+    data_ds = xr.open_dataset(sites_config.filename)
+
+    # Load site generation data
+    metadata_df = pd.read_csv(sites_config.metadata_filename)
+
+    metadata_df.set_index("site_id", inplace=True, drop=True)
```

Comment:
```suggestion
    metadata_df = pd.read_csv(sites_config.metadata_filename, index_col="site_id")
```
I *think* this should do the same thing but might need testing

Reply:
thanks